test/integration/targets/nios_smoke_tests/tasks/prepare.yml

```diff
@@ -0,0 +1,12 @@
+- name: create the provider credentials from the cloud script
+  set_fact:
+    nios_provider:
+        host: "{{ lookup('env', 'NIOS_HOST') }}"
+        username: admin
+        password: infoblox
+- debug: var=nios_provider
+
+- name: install the infoblox client lib 
+  pip:
+    name: infoblox-client
+    state: present
```

Comment:
Put this requirement in `test/runner/requirements/integration.cloud.nios.txt` instead.